src/inc/quic_platform_winuser.h

```diff
@@ -31,6 +31,11 @@
 #define WIN32_LEAN_AND_MEAN 1
 #endif
 
+#ifdef QUIC_UWP_BUILD
+#undef WINAPI_FAMILY
+#define WINAPI_FAMILY WINAPI_FAMILY_DESKTOP_APP
+#endif
```

Comment:
I want to see if we can get a better solution for this before we merge this PR.

Reply:
I have an email out asking.